code/game/machinery/bees_apiary.dm

```diff
@@ -183,7 +183,9 @@
 		for(var/obj/machinery/portable_atmospherics/hydroponics/H in view(7, src))
 			if(H.seed && !H.dead && prob(owned_bee_swarms.len * 10))
 				src.nutrilevel++
-				H.nutrilevel++
+				if(H.nutrilevel < 100)
```

Comment:
I actually did some tests on the matter.

Our implementation of Clamp()

``` DM
#define Clamp(x, y, z) 	(x <= y ? y : (x >= z ? z : x))
```

is faster than the following:

``` DM
#define Clamp(x, y, z) min(max(x, z), y)
```